CONTRIBUTING.md

```diff
@@ -47,6 +48,17 @@ rule_files:
   - more-rules.yaml
 ```
 
+This repository contains Go unit tests. Test files are located in the `o11y/exporter` directory with `_tests.go and .go` file names.
```

Comment:
Go test files are located in each sub-directory under `o11y/exporters` directory with `_test.go` file names.